docs/install.md

```diff
@@ -1,20 +1,24 @@
-`vcd-cli` requires Python 3.
+# Installation 
```

Comment:
Can we also add install instructions for windows?

Reply:
Agree but it will need to be a separate pull request unless somebody has instructions I can use now.

Reply:
Anusuya and Aritra use windows.